.gitignore

```diff
@@ -36,3 +36,4 @@ node_modules
 .vendor
 Goopfile.lock
 venv/
+COMMIT
```

Comment:
@Makman2 Is this okay? while doing a release one commit file will have to be manually placed